src-ghc/Pact/ApiReq.hs

```diff
@@ -389,6 +269,39 @@ decodeYaml :: FromJSON b => FilePath -> IO b
 decodeYaml fp = either (dieAR . show) return =<<
                  liftIO (Y.decodeFileEither fp)
 
+returnCommandIfDone :: Bool -> SigData Text -> IO ByteString
```

Comment:
moving these functions breaks the diff, I assume they are unchanged, if so please leave them where they were